README.md

```diff
@@ -224,7 +224,11 @@ export  LD_LIBRARY_PATH=/usr/local/software/intel/oneapi/2022.1/compiler/latest/
 
 ## Testing
 
-cmake also builds a test suite `bin/testNESO`.
+CMake also builds a suite unit tests (`<build_dir>/test/unitTests`)
+and integration tests (`<build_dir>/test/integrationTests`).
+
+A subset of the tests may be run using `ctest` e.g. `/path/to/NESO/build$ ctest -R TestSuiteName.TestName`.
+See the [googletest user guide](http://google.github.io/googletest/) for more info.
```

Comment:
You're kind of mixing `ctest` and googletest stuff here. Might it make more sense to show an example where the test executable is called directly?

Reply:
Updated that just now.